src/backend/marsha/core/api.py

```diff
@@ -227,6 +229,71 @@ def update_state(request):
     return Response({"success": True})
 
 
+@api_view(["POST"])
+def local_s3_upload(request: HttpRequest, uuid=None):
```

Comment:
I would feel more confortable if these views were moved to the test directory. It will be confusing to have them in the middle of production code. You also need much more explanations in the docstring than just "Endpoint to bypass s3 upload", detailing why we do this and how it works.

Reply:
I agree that this view is used only for e2e tests for now, but I think we can improve it to bypass s3 in our dev environments, with a real local upload management.

Reply:
ok. Then at least a separate file? Maybe add a "development" folder?
I think the term "bypassing" is misleading. It's more like "mocking" or "emulating"?

Reply:
I changed the docstring for `mock`

Reply:
@sampaccoud @lunika I moved local_s3_upload to a src/backend/marsha/development/api.py file.
What de you think about doing the same for DevelopmentLTIView in src/backend/marsha/development/views.py ?

Reply:
+1